koch.nim

```diff
@@ -470,7 +470,7 @@ proc temp(args: string) =
   # commit.
   let (bootArgs, programArgs) = splitArgs(args)
   let nimexec = findNim()
-  exec(nimexec & " c -d:debug " & bootArgs & " compiler" / "nim", 125)
+  exec(nimexec & " c --debugger:native " & bootArgs & " compiler" / "nim", 125)
```

Comment:
This does not replace ``-d:debug``!

Reply:
Ok I added `-d:debug` again. Is this ok now?